screens/OrderFormII.js

```diff
@@ -486,10 +546,18 @@ class OrderFormII extends React.Component {
     })
 
     order.lineItems !== undefined && order.lineItems?.sort((a, b) => {
-      let sort = ["OPEN", "IN_PROCESS", "ALREADY_IN_PROCESS", "DELIVERED", "SETTLED"];
+      let sort = ["OPEN", "IN_PROCESS", "ALREADY_IN_PROCESS", "PREPARED", "DELIVERED", "SETTLED"];
       return sort.indexOf(a.state) - sort.indexOf(b.state);
     });
 
+    let isAllPrepared = 0;
```

Comment:
I guess if you just want to check if there is at least one state open items, you can make this a boolean variable.